palm/pilrc/Portfile

```diff
@@ -1,11 +1,10 @@
 PortSystem 1.0
 
 name			pilrc
-version			3.1
-revision		1
+version			3.2
 categories		palm devel
 license			GPL-2+
-maintainers		nomaintainer
+maintainers		@nkorth
```

Comment:
You should add `openmaintainer` unless there is a reason you should be the only one to maintain this port:

```
maintainers		{@nkorth} \
                        openmaintainer
```

Reply:
ideally one should also add an (obfuscated) e-mail address